core/src/apps/cosmos/transaction.py

```
@@ -117,79 +123,80 @@
         self.msgs_item = {}
 
     def display(self, j: Any, level: int, key_prefix: str) -> None:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `j`

(ANN401)